pkg/flagset/sharing.go

```diff
@@ -136,7 +136,7 @@ func SharingWithConfig(cfg *config.Config) []cli.Flag {
 		},
 		&cli.StringFlag{
 			Name:        "public-driver",
-			Value:       "memory",
+			Value:       "json",
```

Comment:
the json driver has a hard dependency on its db file. This might fail a default ocis run.